### ISSUE TITLE
Button to return ozone molecule doesn't appear right away

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecules-and-light#252]
**Test device**
Dell or iPad
**Operating System**
Win 10 or iPadOS 13.3
**Browser**
Chrome or Safari
**Problem description**
For https://github.com/phetsims/QA/issues/461. Not directly an a11y issue, but can have a larger impact on screen users. Noticed on iPadOS VO.
When you are stepping through while paused, the Return Molecule button doesn't appear until several steps after the ozone breaks apart. This makes some sense. However, when doing this using a screen reader, it will say that the button appears on the same step that it says the molecule breaks apart. This is not the case, and it will not appear for some time. If the screen reader user tries to find it without pressing play or step forward several more times (with no indication this is needed) they will not find it.

**Visuals**
![longwaittobutton](https://user-images.githubusercontent.com/41024075/70947541-92a6dd80-2016-11ea-8dfd-07053c8727a0.gif)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Molecules and Light‬
URL: https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-dev.21/phet/molecules-and-light_en_phet.html?ea&a11y&supportsSound
Version: 1.5.0-dev.21 2019-12-13 19:43:51 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.79 Safari/537.36
Language: en-US
Window: 1536x722
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jessegreenberg** [comment]
Good question @KatieWoe. The current behavior is that the "Return Molecule" button becomes visible when the first constituent molecule is removed from the Observation Window.

It seems like in order to support step with screen readers we will need to wait to alert about this button until it is actually on screen. But assigning to @terracoda and @Matthew-phet to review and decide how to handle.

**Comment by Matthew-phet** [comment]
It would make sense to me to have the "Return Molecule" button appear as soon as the molecule breaks apart, to have it line up with the alert and the sound effect of the molecule breaking apart but i am not sure how possible that is.

**Comment by terracoda** [comment]
I noticed in Step view that button is quite delayed.
We could have 2 separate alerts instead. That might be clearer and less likely to get missed.

**Comment by jessegreenberg** [comment]
> It would make sense to me to have the "Return Molecule" button appear as soon as the molecule breaks apart, to have it line up with the alert and the sound effect of the molecule breaking apart but i am not sure how possible that is.

Could certainly implement this but the current behavior was an explicit design decision so we would want to check with @arouinfar before changing.

**Comment by KatieWoe** [comment]
Could you say "Return molecule button will appear soon" and then "appeared" when it does?

**Comment by terracoda** [comment]
@KatieWoe, nice thought. We haven't found the need to describe anticipated changes. I'd prefer not to take that route.

Something more general is possible, though..."Look for Return Molecule button."
We will continue to optimize this interaction and how it is described.

**Comment by terracoda** [comment]
We decided on changing "Return Molecule" to "New Molecule" in the button's label, and to provide a context response that fires upon appearance of the button. This context response offers a help-text-like interaction hint: "Look for New Molecule button."

@jessegreenberg, these decisions are recorded in the design doc, as well.
Back to you for implementation.

**Comment by terracoda** [comment]
@jessegreenberg, note that the context response for the button is now separate from other context responses about the molecule breaking apart, so it can be timed with the actual appearance of the button.

**Comment by jessegreenberg** [comment]
This issue is now stale, the button label has changed, and the alerts fire when the button appears in the observation window, not when the molecule breaks apart. Closing.
